client/dashboard/store-performance/index.js

```diff
@@ -199,7 +199,7 @@ export default compose(
 
 		if ( statKeys.length === 0 ) {
 			return {
-				hiddenIndicators,
+				hiddenIndicators: [],
```

Comment:
I think this should be renamed to `hiddenBlocks`, I guess I messed it up when merging master into #2148. In that PR I unified `hiddenCharts`, `hiddenIndicators`, etc. to use the same prop name (`hiddenBlocks`) so props would be consistent between sections.
```suggestion
				hiddenBlocks,
```